lib/bracket-matcher.coffee

```diff
@@ -81,7 +82,7 @@ class BracketMatcher
       @editor.moveRight()
       false
     else if autoCompleteOpeningBracket
-      @editor.insertText(text + pair)
+      @origEditorInsertText(text + pair)
       @editor.moveLeft()
```

Comment:
Ok I've digg into adviceBefore, it's like an event system
https://github.com/atom/underscore-plus/blob/master/src/underscore-plus.coffee#L57-L61

wouldn't just not doing the advice / commenting [the line](https://github.com/jacekkopecky/bracket-matcher/blob/fix-166/lib/bracket-matcher.coffee#L37) do the same thing ?